packages/backend/src/config/env.ts

```diff
@@ -1,5 +1,8 @@
 import { z } from 'zod'
 
+import * as dotenv from 'dotenv'
+dotenv.config({})
```

Comment:
Rico uses it because he does not use the docker containers while developing. 
This code should be removed, he should only use it on his local machine, in which he starts the backend locally, not with docker.